### ISSUE TITLE
twig-lint path

### ISSUE THREAD

**Issue by ghost** [lexcast/linter-twig#3]
This may be a silly question but I have twig-lint installed via composer to ~/.composer/vendor/bin. When I try to use $HOME/.composer/vendor/bin in my path or in the Atom twig-lint Path, I get a spawn twig-lint ENOENT error because it seems like it can't handle the $HOME alias (even though when I do which twig-lint on the command line, it finds it fine. 

Is this a limitation of Atom? It's not a big deal as I can symlink to /usr/local/bin but I was just curious. The reason why I use $HOME is because I sync Atom settings from computer to computer and the username might not always be the same.

**Comment by lexcast** [comment]
Yes, the $HOME variable is not recognized. I think is a limitation of the [atom-linter](https://github.com/AtomLinter/atom-linter) helper and its exec method.

**Comment by riley-martine** [comment]
In case this helps someone else, for the path, just put the absolute path to twig-lint (in my case, /home/username/.composer/vendor/bin/twig-lint) in the settings for linter-twig rather than messing around with PATH variables. Despite being able to execute twig-lint from the command line, the package had trouble finding them.

**Comment by MehGokalp** [comment]
Link twig-lint executable to /usr/local/bin by
`ln -s ~/.composer/vendor/asm89/twig-lint/bin/twig-lint /usr/local/bin/twig-lint`
It'll solve the problem.

Note: I'am working on mac os Sierra, do not forget to edit paths to yours.

**Comment by RyanGittins** [comment]
@MehGokalp Thank you!  That symlink is exactly what I needed and was surprisingly hard to find.  For future searchers, the above symlink fixes the following error in Atom:

    Failed to spawn command twig-lint. Make sure twig-lint is installed and on your PATH

**Comment by bgrrtt** [comment]
The `${HOME}` variable worked for me!

```
export PATH="${PATH}:${HOME}/.composer/vendor/bin"
```
